src/QMCWaveFunctions/einspline_helper.hpp

```diff
@@ -361,7 +361,7 @@ namespace qmcplusplus
     mpi::bcast(*myComm,nbands);
 
     //buffer to serialize BandInfo
-    PooledData<RealType> misc(nbands[0]*5);
+    PooledData<double> misc(nbands[0]*5);
```

Comment:
Why double and not a parameterized type?

Reply:
This is to pack send/recv data for the builder. Safe to use highest precision. I will change it to the full precision macro.